mantle/kola/register/register.go

```diff
@@ -112,6 +112,8 @@ type Test struct {
 	// Conflicts is non-empty iff nonexclusive is true
 	// Contains the tests that conflict with this particular test
 	Conflicts []string
+
+	Nvme bool
```

Comment:
To add the `nvme` you still need to touch the `mantle/kola/harness.go` to make use of it.

Reply:
is this what you mean?
https://github.com/coreos/coreos-assembler/pull/3531/files#diff-9db99bb7fefa41cf89f75a965efa109bc8349ab0a55fdb093fa46775724791cbR1008

or am i missing something else?